### PR TITLE
Fix markdown formatting in docs

### PR DESCRIPTION
**Pull request by iskhakov**
# What this PR does

This PR adds `--fix` flag to `markdownlint` pre-commit command and fixes existing formatting to comply with markdown formatting rules.

## Checklist

- [ ] Tests updated
- [x] Documentation added
- [ ] `CHANGELOG.md` updated

### CODE REVIEW

**Review by joeyorlando — APPROVED** [review]
nice! good idea

File: docs/sources/calendar-schedules/ical-schedules/index.md [hunk 1]
```diff
@@ -14,52 +14,63 @@ weight: 300
...
 
-1. Create a new calendar in your calendar app, then review and adjust default settings as needed. 
-1. In your new calendar, create events that represent on-call shifts. You must use Grafana usernames as the event title to associate users with each shift.
-1. Once your on-call calendar is complete, go to your calendar settings to locate the secret iCal URL. For example, in a Google calendar, this URL can be found in **Settings** > **Settings for my calendars** > **Integrate calendar** > **Secret address in iCal format**.
+1. Create a new calendar in your calendar app, then review and adjust default settings as needed.
+2. In your new calendar, create events that represent on-call shifts. You must use Grafana usernames as the event title
+3. to associate users with each shift.
+4. Once your on-call calendar is complete, go to your calendar settings to locate the secret iCal URL. For example, in
+5. a Google calendar, this URL can be found in **Settings** > **Settings for my calendars** > **Integrate calendar** >
+6. **Secret address in iCal format**.
```
Comment (alyssawada) [inline docs/sources/calendar-schedules/ical-schedules/index.md:48]: @iskhakov @joeyorlando it looks like the markdown formatting messed up ordered lists starting new lines as another numbered step. Also we use repetitive list numbering so these should remain as:
1.
1.
1.

File: docs/sources/calendar-schedules/web-schedule/_index.md [hunk 2]
```diff
@@ -12,42 +12,50 @@ keywords:
...
-- **Slack channel:** Choose a primary Slack channel to send notifications about on-call shifts, such as unassigned on-call shifts.
+- **Slack channel:** Choose a primary Slack channel to send notifications about on-call shifts, such as unassigned
+- on-call shifts.
 - **Slack user group:** Choose a Slack user group to receive current on-call updates.
 - **Notification frequency:** Specify whether or not to send shift notifications to scheduled team members.
-- **Action for slot when no one is on-call:** Define how your team is notified when an empty shift causes a gap in on-call coverage.
+- **Action for slot when no one is on-call:** Define how your team is notified when an empty shift causes a gap in
+- on-call coverage.
 - **Current shift notification settings:** Select how users are notified when their on-call shift begins.
 - **Next shift notification settings:** Specify how users are notified of upcoming shifts.
```
Comment (alyssawada) [inline docs/sources/calendar-schedules/web-schedule/_index.md:42]: Looks like unordered lists also got messed up a bit here with new lines

File: docs/sources/calendar-schedules/web-schedule/_index.md [hunk 3]
```diff
@@ -12,42 +12,50 @@ keywords:
...
+The schedule view is a detailed calendar representation of your on-call schedule. It contains three interactive weekly
+calendars and a 24-hour on-call status bar for visualizing who’s on-call and what time it is for your teammates.
 
 Understand your schedule view:
 
 - **Final schedule:** The final schedule provides a combined view of rotations and overrides
 - **Rotations:** The rotations calendar represents all recurring on-call rotations for a given schedule.
-- **Overrides:** The override calendar represents temporary adjustments to the recurring on-call schedule. Any events on this calendar will take precedence over the rotations calendar.
+- **Overrides:** The override calendar represents temporary adjustments to the recurring on-call schedule. Any events
+- on this calendar will take precedence over the rotations calendar.
```
Comment (alyssawada) [inline docs/sources/calendar-schedules/web-schedule/_index.md:54]: Unordered list incorrectly formatter here as well

File: docs/sources/calendar-schedules/web-schedule/calendar-export/index.md [hunk 4]
```diff
@@ -15,53 +15,60 @@ weight: 500
...
+rest of your schedule.  
 
 There are two schedule export options available:
 
-- **On-call schedule export** - Exports all on-call shifts for a particular schedule, including rotations, overrides, and assigned users.
-- **User-specific schedule export** - Exports assigned on-call shifts for a particular user. Use this export option to add your assigned on-call shifts to your calendar. 
+- **On-call schedule export** - Exports all on-call shifts for a particular schedule, including rotations, overrides,
+- and assigned users.
+- **User-specific schedule export** - Exports assigned on-call shifts for a particular user. Use this export option to
+- add your assigned on-call shifts to your calendar.
```
Comment (alyssawada) [inline docs/sources/calendar-schedules/web-schedule/calendar-export/index.md:27]: Unordered list formatting impacted

File: docs/sources/calendar-schedules/web-schedule/calendar-export/index.md [hunk 5]
```diff
@@ -15,53 +15,60 @@ weight: 500
...
 1. In Grafana OnCall, navigate to the **Schedules** tab.
-1. Open the schedule you’d like to export by clicking on the schedule name.
-1. Click **Export** in the upper right corner, then click **+ Create iCal link** to generate a secret iCal URL.
-1. Copy the iCal link and store it somewhere you’ll remember. Once you close the schedule export window, you won't be able to access the iCal link. 
-1. Open your calendar settings to add a calendar from a URL (This step varies based on your calendar app).
+2. Open the schedule you’d like to export by clicking on the schedule name.
+3. Click **Export** in the upper right corner, then click **+ Create iCal link** to generate a secret iCal URL.
+4. Copy the iCal link and store it somewhere you’ll remember. Once you close the schedule export window, you won't be
+5. able to access the iCal link.
+6. Open your calendar settings to add a calendar from a URL (This step varies based on your calendar app).
```
Comment (alyssawada) [inline docs/sources/calendar-schedules/web-schedule/calendar-export/index.md:43]: Ordered list improperly formatted and repetitive list numbering was replaced.

File: docs/sources/calendar-schedules/web-schedule/calendar-export/index.md [hunk 6]
```diff
@@ -15,53 +15,60 @@ weight: 500
...
-1. Copy the iCal link and store it somewhere you’ll remember. Once you close your user profile, you won't be able to access the iCal link again.
-1. Open your calendar settings to add a calendar from a URL (This step varies based on your calendar app).
-
-## Revoke an iCal export link 
+2. Click **View my profile** in the upper right corner.
+3. From the **User Info** tab, navigate to the iCal link section.
+4. Click **+ Create iCal link** to generate your secret iCal URL.
+5. Copy the iCal link and store it somewhere you’ll remember. Once you close your user profile, you won't be able to
+6. access the iCal link again.
+7. Open your calendar settings to add a calendar from a URL (This step varies based on your calendar app).
```
Comment (alyssawada) [inline docs/sources/calendar-schedules/web-schedule/calendar-export/index.md:57]: Ordered list improperly formatted and repetitive list numbering was replaced.

File: docs/sources/calendar-schedules/web-schedule/create-schedule/index.md [hunk 7]
```diff
@@ -35,35 +39,36 @@ To create a new on-call schedule:
...
-You can separate users into user groups to rotate through individual users per shift. User groups that contain multiple users results in all users in the group being included in corresponding shifts.
-1. When you’re satisfied with the rotation preview, click **Create**.
-
-
-[comment]: <> (For further instruction on rotation configuration, refer to Manage and configure rotations)
+2. Complete the rotation creation form according to your rotation parameters.
+3. Add users to the rotation from the dropdown.
+You can separate users into user groups to rotate through individual users per shift. User groups that contain
+4. multiple users results in all users in the group being included in corresponding shifts.
+5. When you’re satisfied with the rotation preview, click **Create**.
```
Comment (alyssawada) [inline docs/sources/calendar-schedules/web-schedule/create-schedule/index.md:56]: Ordered list format impacted here and repetitive list numbering was replaced.

**Review by alyssawada — COMMENTED** [review]
The markdown formatting introduced format issues with numerical ordered lists and unordered lists. Adding new lines resulted in additional steps or bullet points containing fragmented sentences. As well removing repetitive list formatting from ordered lists

**Pull request merged**